### PR TITLE
Update change history of ShardingSphere-Proxy.

### PR DESCRIPTION
**Pull request by yx9o**
Update change history of ShardingSphere-Proxy.

### CODE REVIEW

File: docs/document/content/reference/api-change-history/shardingsphere-proxy/_index.en.md [hunk 1]
```diff
@@ -4,6 +4,62 @@ weight = 1
...
+    users:
+      - root@localhost:root # <username>@<hostname>:<password>
+      - sharding@:sharding
+    provider:
+      type: NATIVE # Must be explicitly specified.
+ ```
+
+If the hostname is % or empty, it means no restrict to the user’s host.
+
+The type of the provider must be explicitly specified. Refer to [5.11 Proxy](https://shardingsphere.apache.org/document/current/en/dev-manual/proxy/) for more implementations.
```
Comment (RaigorJiang) [inline docs/document/content/reference/api-change-history/shardingsphere-proxy/_index.en.md:45]: The link should no related to `current` version.

File: docs/document/content/reference/api-change-history/shardingsphere-proxy/_index.cn.md [hunk 2]
```diff
@@ -4,6 +4,62 @@ weight = 1
...
+    users:
+      - root@localhost:root  # <username>@<hostname>:<password>，hostname 为 % 或空字符串，则代表不限制 host。
+      - sharding@:sharding
+    provider:
+      type: NATIVE  # 必须显式指定
+```
+
+hostname 为 % 或空字符串，则代表不限制 host。
+
+provider 的 type 必须显式指定，具体实现可以参考 [5.11 Proxy](https://shardingsphere.apache.org/document/current/cn/dev-manual/proxy/)
```
Comment (RaigorJiang) [inline docs/document/content/reference/api-change-history/shardingsphere-proxy/_index.cn.md:45]: The link should no related to current version.

File: docs/document/content/reference/api-change-history/shardingsphere-proxy/_index.en.md (hunk 1 above)
Comment (yx9o) [reply docs/document/content/reference/api-change-history/shardingsphere-proxy/_index.en.md]: > 

Done.

**Review by RaigorJiang — APPROVED** [review]

**Comment by RaigorJiang** [comment]
@yx9o  Thank you~

**Pull request merged**